sqlglot/generator.py

```diff
@@ -3860,9 +3860,16 @@ def _replace_line_breaks(self, string: str) -> str:
     def copyparameter_sql(self, expression: exp.CopyParameter) -> str:
         option = self.sql(expression, "this")
 
-        if option.upper() == "FILE_FORMAT":
-            values = self.expressions(expression, key="expression", flat=True, sep=" ")
-            return f"{option} = ({values})"
+        if isinstance(expression.expression, list):
+            upper = option.upper()
+
+            # Snowflake FILE_FORMAT
```

Comment:
This comment doesn't explain the next line well. Add some more context to it, eg. "Snowflake's FILE_FORMAT property uses a space separator, while Databricks' uses commas".